src/nvim/option.c

```diff
@@ -3806,8 +3744,10 @@ void set_option_direct(OptIndex opt_idx, OptVal value, int opt_flags, scid_T set
   const bool scope_both = (opt_flags & (OPT_LOCAL | OPT_GLOBAL)) == 0;
   void *varp = get_varp_scope(opt, scope_both ? OPT_LOCAL : opt_flags);
 
-  set_option(opt_idx, varp, optval_copy(value), opt_flags, set_sid, true, true, errbuf,
-             sizeof(errbuf));
+  const char *errmsg = set_option(opt_idx, varp, optval_copy(value), opt_flags, set_sid, true, true,
+                                  errbuf, sizeof(errbuf));
+  assert(errmsg == NULL);
+  (void)errmsg;  // ignore unused warning
```

Comment:
nit: this is very common practice, so could drop the comment

Reply:
> nit (or just future reference): this is very common practice, so could drop the comment

Understood, though fwiw other places in the codebase that do this also have this comment, which is why I added it